api_app/views.py

```diff
@@ -697,9 +697,15 @@ def get_queryset(self):
             user=self.request.user, organization__isnull=False
         )
         if membership.exists():
-            result |= PluginConfig.objects.filter(
-                organization=membership[0].organization,
+            user_membership = membership[0]
```

Comment:
if you are using the `[0]`, you can use `try/catch Membership.objects.get(query)`